### PR TITLE
hold colors on form

### PR DESCRIPTION
**Pull request by raymondng1**
1. Run App
2. Go to form and fill it out
3. If you go back, the hold color css styles are now mapped to their string counterpart

### CODE REVIEW

File: app/components/CreateRouteHoldsColor.js [hunk 1]
```diff
@@ -16,8 +18,15 @@ class CreateRouteHoldsColor extends Component {
       h.style.backgroundColor = color;
     });
   }
+  checkHoldColor = holdColor => {
+    for (let key in holdColorDictonary) {
+      return holdColorDictonary[holdColor];
```
Comment (nries1) [inline app/components/CreateRouteHoldsColor.js]: A return statement will break the for loop and return you out of the function, so this is always going to return after the first iteration of the for loop

Comment (raymondng1) [reply app/components/CreateRouteHoldsColor.js]: I just wanted your insight of why we don't want to break out of the for loop and return out of the function after the iteration? My understanding is that the state (routeModel.holdColor) is changing when selecting the option which will run function and break out of the loop when the **value[key]** is found. 

The function will re-run again a new iteration will begin? and breakout of the loop when a different option selected has been chosen.

Comment (raymondng1) [reply app/components/CreateRouteHoldsColor.js]: @nries1 Hey I actually don't think we need a for loop and could just write the function cleaner as:

  **checkHoldColor = holdColor => {
    return holdColorDictonary[holdColor]
  };**

Comment (nries1) [reply app/components/CreateRouteHoldsColor.js]: Ye, you don't need a for loop. You don't even need a function really. All you need to do is look up the routeModel.holdColor in the holdColorDictionary

**Review by nries1 — APPROVED** [review]
Thanks for this!

File: app/components/CreateRouteHoldsColor.js (hunk 1 above)
Comment (raymondng1) [reply app/components/CreateRouteHoldsColor.js]: Thanks, sorry I was just overthinking this way too much.....

Comment (nries1) [reply app/components/CreateRouteHoldsColor.js]: haha. no problem. It's fine to use a function. Just don't need the for loop

**Pull request merged**